jsk_kinova_robot/jsk_kinova.rosinstall

```diff
@@ -0,0 +1,11 @@
+# ros_kortex is updating to v2.3.X in master branch
+# but the internal firmware in JSK kinova is still ver2.2.0
+# we need to use it until firmware is updated.
```

Comment:
if you have forked branch, please make PR for your custom commits, for example
https://github.com/708yamaguchi/ros_kortex/commit/9b5105aea142e542cfb4fdf7ac7cd2ec2cf91b98

Reply:
@k-okada 
In fact, this custom commit already became a PR and was merged:
https://github.com/Kinovarobotics/ros_kortex/pull/162

However, this was merged after v2.3.0 was released.
So when we tagged to the merge commit of that PR, v2.3.0 was forcibly introduced.
Also, tagging to that custom commit itself didn't work (sorry, but I don't remember why, @708yamaguchi may know).

Reply:
So if we want to use origin branch, we have to update firmware in JSK kinova.
We don't know much about pros and cons of updating.
(If my memory is correct, @708yamaguchi says that downgrading firmware is not supported, so we have a point of no return here.)

Reply:
@pazeshun 

Thank you very much for your explanation.

Actually, I created release request of v2.2.x kortex_ros + prefix feature
(namely, release request of https://github.com/708yamaguchi/ros_kortex/commit/9b5105aea142e542cfb4fdf7ac7cd2ec2cf91b98 ).

However, that was not accepted...
https://github.com/Kinovarobotics/ros_kortex/issues/187